app/views/my/password.html.erb

```diff
@@ -26,16 +26,14 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
-
 <% html_title l(:label_my_account), l(:button_change_password) %>
 <%= toolbar title: l(:button_change_password) %>
 
 <%= error_messages_for 'user' %>
 <%= styled_form_tag({:action => :change_password},
-                    {:autocomplete => 'off'}) do %>
+                    {:autocomplete => 'off', class: 'change_password -higher'}) do %>
```

Comment:
I was just wondering what `-higher` could mean, just to find out that this leads to the already discussed `min-height` hack.

This really smells like a quick solution that won't work out in the general use case.